pkg/log/log.go

```diff
@@ -65,6 +65,16 @@ func ParseVerbosityLevel(s string) (Level, error) {
 	return Level(i), err
 }
 
+// MustParseVerbosityLevel returns a verbosity Level parsed from the given s.
+// It panics if the given s is not a valid verbosity level.
+func MustParseVerbosityLevel(s string) Level {
+	l, err := ParseVerbosityLevel(s)
+	if err != nil {
+		panic(err)
```

Comment:
please handle error instead of panicing

Reply:
This is the `Must` pattern commonly used in the std lib (e.g. `regexp.MustCompile`). It is used in cases where the input has already been validated and simplifies initialization.

Reply:
Regardless of naming convention, having panic in code may lead to unexpected program termination. It is risky to be certain that some execution flows will not cause program to terminate.

Also, this rule is documented in [coding style](https://github.com/ethersphere/bee/blob/master/CODINGSTYLE.md#panic-with-care) doc.